### PR TITLE
Begin of support for mingw

### PR DESCRIPTION
**Pull request by ImperatorS79**

### CODE REVIEW

File: xmake.lua [hunk 1]
```diff
@@ -24,7 +24,7 @@ local modules = {
 	Network = {
 		Deps = {"NazaraCore"},
 		Custom = function()
-			if is_plat("windows") then 
+			if is_plat("windows") or is_plat("mingw") then
```
Comment (SirLynix) [inline xmake.lua]: use
```lua
is_plat("windows", "mingw")
```
as a shortcut

Comment (ImperatorS79) [reply xmake.lua]: Okay

**Comment by SirLynix** [comment]
It would be great to add a MinGW CI to test it.

**Comment by ImperatorS79** [comment]
You could do a MinGW-build.yml which install msys2 and the needed pacman packages, then build.

**Pull request merged**